docs/onboarding_to_cluster.md

```diff
@@ -0,0 +1,151 @@
+# Onboarding to a cluster
```

Comment:
Can we add a brief summary of what this doc accomplishes, should be a short one or two liner.